templates/workloads/services/lb-web/cf.yml

```
@@ -36,6 +36,8 @@ Parameters:
     Type: String
   TargetPort:
     Type: Number
+  Stickiness:
+    Type: String
```

Can we add a default too so that we safe-guard ourselves against situations where the param is not provided:
```suggestion
  Stickiness:
    Type: String
    Default: false
```

---

I just added.